### DISCUSSION TITLE
Mention required Intents in the Guide

### DISCUSSION THREAD

**Discussion by PiotrWieczorek98** [discordjs/discord.js#6661] [Ideas]
Hello,
After wasting half a day debugging what's wrong with my audio player I found the answer on discord discussion - turns out I didn't have required intent Intents.FLAGS.GUILD_VOICE_STATES in my client. I believe there's more people like me so I would be thankful if you add mention about required Intent in the Audio Player guide.

**Comment by ImRodry** [comment]
There is a page on Discord's API docs about intents and the events they enable. You can see this link by doing /tag intents on the Discord server. You should know that v13 requires intents and do the proper research to see what intents are needed for what events

**Reply by PiotrWieczorek98** [reply]
I started using  discord.js  after using discord.py, when it was already on v13 so I didn't know what intent even is, as the guide mentions it only in the beginning as "The Intents.FLAGS.GUILDS intents option is necessary for your client to work properly."

I just suggest to add a mention about this in the Voice section (https://discordjs.guide/voice/#installation)
